.github/workflows/reviewdog.yml

```
@@ -0,0 +1,35 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  textlint:
+    name: runner / textlint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Setup node/npm
+        uses: actions/setup-node@v1
+        with:
+          node-version: '15'
```

Node.jsのLTSは16なので、16を推奨したいですね
```suggestion
          node-version: '16'
```

---

:iine: